stac_model/pyproject.toml

```diff
@@ -0,0 +1,202 @@
+
+# Poetry pyproject.toml: https://python-poetry.org/docs/pyproject/
+[build-system]
+requires = ["poetry_core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+
+[tool.poetry]
+name = "stac-model"
+version = "0.1.1.alpha2"
+description = "A PydanticV2 validation and serialization libary for the STAC ML Model Extension"
+readme = "README.md"
+authors = ["Ryan Avery <ryan@wherobots.com>"]
+license = "Apache Software License 2.0"
+repository = "https://github.com/rbavery/stac-model"
+homepage = "https://github.com/rbavery/stac-model"
+packages = [
+  {include = "stac_model"}
+]
+
+
+# Keywords description https://python-poetry.org/docs/pyproject/#keywords
+keywords = []  # UPDATEME with relevant keywords
+
+
+# Pypi classifiers: https://pypi.org/classifiers/
+classifiers = [  # UPDATEME with additional classifiers; remove last classifier to allow publishing on PyPI
+  "Development Status :: 3 - Alpha",
+  "Intended Audience :: Developers",
+  "Operating System :: OS Independent",
+  "Topic :: Software Development :: Libraries :: Python Modules",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+]
```

Comment:
Add the following:
- `Framework :: Pydantic`
- `Framework :: Pydantic :: 2`
- `Intended Audience :: Information Technology`
- `Intended Audience :: Science/Research`
- `Topic :: File Formats :: JSON :: JSON Schema`
- `Topic :: Scientific/Engineering :: Artificial Intelligence`

maybe others https://pypi.org/classifiers/

Reply:
good tip, I added

```
  "Topic :: Scientific/Engineering :: GIS",
  "Topic :: Scientific/Engineering :: Image Processing",
  "Topic :: Scientific/Engineering :: Image Recognition",
```